### PR TITLE
PascalCasing on component filenames

### PR DESCRIPTION
**Pull request by teallarson**
## Change description

Front end component filenames are now in PascalCase to match React's name casing convention. 

### Development

- [ ] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: ui/ui-components/pages/logs/list.tsx [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-
-      <LogsTabs active="list" />
-      <LogsList {...props} />
-    </>
-  );
-}
-
-Page.getInitialProps = async (ctx) => {
-  return LogsList.hydrate(ctx);
-};
```
Comment (evantahler) [inline ui/ui-components/pages/logs/list.tsx]: Where did this page go?

File: ui/ui-components/pages/logs/stream.tsx [hunk 2]
```diff
@@ -1,20 +0,0 @@
...
-
-      <LogsTabs active="stream" />
-      <LogsStream {...props} />
-    </>
-  );
-}
-
-Page.getInitialProps = async (ctx) => {
-  return LogsStream.hydrate(ctx);
-};
```
Comment (evantahler) [inline ui/ui-components/pages/logs/stream.tsx]: Where did this page go?

**Review by evantahler — CHANGES_REQUESTED** [review]
Dang that's a lot of changes!

There were 2 pages removed:
* ui/ui-components/pages/logs/stream.tsx
* ui/ui-components/pages/logs/stream.tsx

I think they are still in use though:

Screen Shot 2021-11-10 at 8 25 09 AM

**Comment by teallarson** [comment]
@evantahler back now!  That was bizarre.  I reset the commit and undid those changes.

**Review by evantahler — APPROVED** [review]
Well, things looks good now!

**Pull request merged**